spec/features/tags_spec.rb

```diff
@@ -86,6 +86,8 @@ def n_tagables(n, tagable_category)
           expect(page).to have_text tag.name
           expect(page).to have_text tag.description
         end
+
+        it "shows edit tab"
```

Comment:
left unimplemented... oversight?

Reply:
ah yeah, ended up the tests to a different part of the file.